spec-tests/src/test/resources/settings.properties

```diff
@@ -1,4 +1,4 @@
-spec.test.client.host=localhost
+spec.test.client.host=candlepin-el8.example.com
```

Comment:
I think this change is okay, but is there a reason why the host name needed to be updated?

Reply:
no, I have to change it to run it locally, then remember to flip it back so it will run the tests on Jenkins.